datavault-webapp/pom.xml

```diff
@@ -134,6 +133,22 @@
             <artifactId>commons-fileupload</artifactId>
             <version>1.3</version>
         </dependency>
+        
+        <!--  Selenium -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+        </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
```

Comment:
We should remove this dependency if we are not using it.

Reply:
Which dependency do you mean? `org.testng`? It is used - everything in `LoginFT` & `DepositFT` uses TestNG annotations. I could rewrite them just using Junit if that's what you've done elsewhere and you think it would be better

Reply:
No sorry, I misunderstood what 'TestNG' was doing, I though you were not using it any-more!